versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.55"
+      version = ">= 4.6"
```

Comment:
since this is a breaking change, lets update the min provider version to `5.0` and min Terraform version to `1.0`